pkg/smartcontract/rpcbinding/binding.go

```diff
@@ -334,9 +334,17 @@ func scTemplateToRPC(cfg binding.Config, ctr ContractTmpl, imports map[string]st
 	for i := range ctr.SafeMethods {
 		switch ctr.SafeMethods[i].ReturnType {
 		case "interface{}":
-			imports["github.com/nspcc-dev/neo-go/pkg/vm/stackitem"] = struct{}{}
-			ctr.SafeMethods[i].ReturnType = "stackitem.Item"
-			ctr.SafeMethods[i].CallFlag = "Item"
+			abim := cfg.Manifest.ABI.GetMethod(ctr.SafeMethods[i].NameABI, len(ctr.SafeMethods[i].Arguments))
+			if abim.ReturnType == smartcontract.InteropInterfaceType {
```

Comment:
I'm a little warried about it, the case is that we can't reliably distinguish iterator from some other interop interface. We probably need to add a note to generator documentation.

Reply:
> Isn't it already documented now? 

Huh, turns out it's not. For some reason I was sure it is, maybe forgot to commit something, will fix.

Reply:
Yup, forgot to commit that, fixed.